<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorPageWidget</class>
 <widget class="QWidget" name="MonitorPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="verticalLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QToolButton" name="columnBtn">
           <property name="toolTip">
            <string>Настройка видимости столбцов</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appclient.icons.qrc">
             <normaloff>:/meteo/appclient/icons/settings.png</normaloff>:/meteo/appclient/icons/settings.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="tableBtn">
           <property name="toolTip">
            <string>Табличный вид</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appclient.icons.qrc">
             <normaloff>:/meteo/appclient/icons/table.png</normaloff>:/meteo/appclient/icons/table.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="updateBtn">
           <property name="toolTip">
            <string>Поиск сервисов</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appclient.icons.qrc">
             <normaloff>:/meteo/appclient/icons/readdata.png</normaloff>:/meteo/appclient/icons/readdata.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="restartBtn">
           <property name="toolTip">
            <string>Перезапустить процессы</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="appclient.icons.qrc">
             <normaloff>:/meteo/appclient/icons/restart_red.png</normaloff>:/meteo/appclient/icons/restart_red.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableProcs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>5</width>
           <height>5</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>1</number>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Приложение</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
          <property name="toolTip">
           <string>Состояние</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
          <property name="toolTip">
           <string>Статус</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Файл</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>СПО</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>test</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>test</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>22929292</string>
          </property>
         </item>
         <item row="0" column="5">
          <property name="text">
           <string>test</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>219</red>
            <green>219</green>
            <blue>219</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>219</red>
            <green>219</green>
            <blue>219</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>219</red>
            <green>219</green>
            <blue>219</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>219</red>
            <green>219</green>
            <blue>219</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="appclient.icons.qrc"/>
 </resources>
 <connections/>
</ui>
